sprocs/check_join_group.sql

```diff
@@ -0,0 +1,70 @@
+CREATE OR REPLACE FUNCTION
+    check_join_group (
```

Comment:
For sprocs we are trying to name them starting with the table they mainly affect. Based on that, could we call this `group_users_insert` or something like that?